composer.json

```diff
@@ -139,11 +148,36 @@
   ],
   "archive": {
     "exclude": [
-      "node_modules",
-      "coverage",
-      "homestead",
-      "build",
-      ".git"
+      "!/.env",
```

Comment:
Consider doing the reverse, first exclude all and then only add the files needed.

Reply:
I will remember this for next time ok?